### DISCUSSION TITLE
Read Account Tokens

### DISCUSSION THREAD

**Discussion by PapitoMyKing** [phantom/docs#202] [General]
Hi,

I would like to know which NFTs an account has. In the developer docs there is only connecting and sending transactions.

This is what I'm looking at:
https://docs.phantom.app/solana/integrating-phantom

Thanks!

**Comment by adamdelphantom** [comment]
Hi @PapitoMyKing!

In order to retrieve token accounts and balances you will need to call an rpc server using the connected wallets address which you can get from the Phantom provider. 

There are many options for solana rpc services. [Helius](https://www.helius.dev) offers a free tier to get started, among others.

**Reply by Radcastersignal** [reply]
Hi @adamdelphantom 
I need your help, I created a solana nft collection befor one year ago but I didn't make sales from the mint just 300/2345 minted at that time.
Now I want to close all the project accounts and recover the sol rent balance or any balance may can recoverd but I don't have all the files that I generat and creat the project , I save just the keypair.json file and the addresses of the candy machine, Metadata and mint address. How can I close the project accounts and recover some SOL balance?
Pls I need gaudline or any toterial to do that
